PyInstaller/building/datastruct.py

```diff
@@ -50,7 +50,7 @@ def _normentry(self, entry):
             logger.info("TOC found a %s, not a tuple", entry)
             raise TypeError("Expected tuple, not %s." % type(entry).__name__)
         name, path, typecode = entry
-        if typecode in ["BINARY", "DATA"]:
+        if typecode in ["BINARY", "DATA"] and not name.endswith('.typelib'):
             # Normalize the case for binary files and data files only (to avoid duplicates
```

Comment:
@htgoebel I don't know exactly why it can't find it but I do know that when importing an introspected module and requiring a specific version with `gi.require_version('Gst', '1.0')` it tries to load the typelib corresponding to the first argument in this case `Gst` which fails if the typelib is named `gst-1.0.typelib` instead of `Gst-1.0.typelib`.

Would using a regex that only matches the typelib naming pattern be better? Some thing like `.*?\-.\..\.typelib`?